#goal, empty
---
#result is Odometry array
nav_msgs/Odometry[] result_odom_array
---
#feedback, empty